micro-docker/intel-flash/Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM debian:jessie
+
+RUN apt-get update && apt-get install --no-install-recommends -y \
+               ipmitool \
+               python \
+               curl \
+               vim-tiny \
+               less \
+               ca-certificates \
+               sudo \
+               ifupdown \
+               lib32z1 \
+               lib32ncurses5 \
+               lib32bz2-1.0 \
+               lib32gcc1 \
+               lib32stdc++6 \
+               unzip \
+               udev \
+    && rm -rf /var/lib/apt/lists/*
+
+ENV NODE_VERSION 4.6.2
```

Comment:
I suggest changing the node version to a build argument (see https://docs.docker.com/engine/reference/builder/#arg) and add description in readme about how to specify the node version, so if RackHD changes the node version, the Dockerfile can keep unchanged.

Reply:
If RackHD changes the node version, we should update Dockerfile. Otherwise, we have to update Readme to inform users about the changes.